website/content/en/blog/_index.md

```diff
@@ -17,6 +17,11 @@ Want to try the latest version of Kubeflow on AWS? Follow the steps in [Kubeflow
 A curated list of blogs illustrating how to build flexible, scalable ML workflows on Kubernetes using AWS distribution of Kubeflow. 
 Find case studies, use cases, best practices, benchmarks, and more.
 
+1. [Enabling hybrid ML workflows on Amazon EKS and Amazon SageMaker with one-click Kubeflow on AWS deployment](https://aws.amazon.com/blogs/machine-learning/enabling-hybrid-ml-workflows-on-amazon-eks-and-amazon-sagemaker-with-one-click-kubeflow-on-aws-deployment/) - In this blog, we discuss Kubeflow on AWS v1.6.1 features and highlight three important integrations that have been bundled on one platform to offer: 
+* Infrastructure as Code (IaaC) one-click solution that automates the end-to-end installation of Kubeflow, including EKS cluster creation
+* Support for distributed training on Amazon SageMaker 
```

Comment:
Support for running machine learning workloads on Amazon SageMaker from Kubeflow